### PR TITLE
improve integration services

### PR DESCRIPTION
**Pull request by huybui38**
//TODO: Make json more robust

### CODE REVIEW

File: integration/plc-integration/src/main/java/com/hbc/pms/plc/io/IoDetailsRepository.java [hunk 1]
```diff
@@ -0,0 +1,11 @@
+package com.hbc.pms.plc.io;
+
+import java.util.List;
+
+public interface IoDetailsRepository {
```
Comment (oHTGo) [inline integration/plc-integration/src/main/java/com/hbc/pms/plc/io/IoDetailsRepository.java:5]: Please use IIoDetailsRepository

File: integration/plc-integration/src/main/java/com/hbc/pms/plc/io/IoDetailsService.java [hunk 2]
```diff
@@ -0,0 +1,12 @@
+package com.hbc.pms.plc.io;
+
+public interface IoDetailsService {
```
Comment (oHTGo) [inline integration/plc-integration/src/main/java/com/hbc/pms/plc/io/IoDetailsService.java:3]: Like above comment

File: integration/plc-integration/src/main/java/com/hbc/pms/plc/io/IoDetailsRepository.java (hunk 1 above)
Comment (huybui38) [reply integration/plc-integration/src/main/java/com/hbc/pms/plc/io/IoDetailsRepository.java:5]: @oHTGo We don't do that here in Java Spring Project. We shouldn't use "I" for interface and "Impl" in our Spring project.

File: integration/plc-integration/src/main/java/com/hbc/pms/plc/io/IoDetailsService.java (hunk 2 above)
Comment (huybui38) [reply integration/plc-integration/src/main/java/com/hbc/pms/plc/io/IoDetailsService.java:3]: Like above response

**Review by zKirinz — APPROVED** [review]

**Pull request merged**